### PR TITLE
Handles empty firewall rule description - BR-2812

### PR DESCRIPTION
**Pull request by ngaumont**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Expired session handling not prompting for new login.
 - `cb list` completion to include `--format`.
+- `cb info` now correctly handles firewall rules with an empty description
```
Comment (abrightwell) [inline CHANGELOG.md:24]: Move this up under `## [Unreleased`:

```
## [Unreleased]

### Fixed
- `cb info` now correctly handles firewall rules with an empty description

## [3.6.0] - 2024-07-26
```

File: spec/cb/firewall_rule_spec.cr [hunk 2]
```diff
@@ -92,15 +92,15 @@ Spectator.describe FirewallRuleList do
       action.output = IO::Memory.new
       action.network_id = network.id
 
-      expect(client).to receive(:get_firewall_rules).and_return [Factory.firewall_rule]
+      expect(client).to receive(:get_firewall_rules).and_return [Factory.firewall_rule(description: nil)]
```
Comment (abrightwell) [inline spec/cb/firewall_rule_spec.cr:95]: Let's keep a test case for both. One with and one without a description. 👍

**Review by abrightwell — COMMENTED** [review]
LGTM. Just a couple comments.

File: CHANGELOG.md (hunk 1 above)
Comment (ngaumont) [reply CHANGELOG.md]: Sorry

**Review by abrightwell — APPROVED** [review]

**Pull request merged**